libminifi/include/core/Property.h

```diff
@@ -96,7 +96,7 @@ class Property {
   std::string getDisplayName() const;
   std::vector<std::string> getAllowedTypes() const;
   std::string getDescription() const;
-  std::shared_ptr<PropertyValidator> getValidator() const;
+  gsl::not_null<const PropertyValidator*> getValidator() const;
```

Comment:
I would return a `const PropertyValidator&` here, too, for simplicity.

Reply:
also done in 1a816b300b0dd3be28f8a5c55188893fca238159